staff/manuelbarzi/cv/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Manu Wave - CV</title>
+</head>
+<body>
+    <h1>Manu Wave - CV</h1>
+
+    <p>I think I'm a good person, but sometimes I get angry too easily, but... well, it doesn't matter, does it?</p>
+
+    <h2>Position</h2>
+
+    <h3>Trainer at SkyLab Coders Academy</h3>
+
+    <h2>Past</h2>
+
+    <h3>Java &amp; Javascript Enginner at WTF</h3>
+
+    <h4>Java &amp; Javascript Enginner at Ortivamedia</h4>
+
+    <h3>Java &amp; Javascript Enginner at OMG</h3>
+
+    <h4>Java &amp; Javascript Enginner at Super Chachi Co</h4>
+
+    <h3>...</h3>
+    
+    <h2>Degrees</h2>
+
+    <h3>Robotics Enginnering (2003)</h3>
+
+    <h3>Electronics Enginnering (1999)</h3>
+
+    <h2>Languages</h2>
+
+    <ul>
+        <li>Spanish (native)</li>
+        <li>Catalan (C level)</li>
+        <li>English (nice)</li>
+    </ul>
+
+    <h2>Hobbies</h2>
+
+    <ul>
+        <li>Resting...</li>
+        <li>Watching the sky...</li>
+        <li>Drinking coffee...</li>
+    </ul>
+</body>
+</html>
```

Comment:
"no newline at the end of line" automatic message appears, not sure why

Reply:
@Melinx that's not a problem, don't worry ,)

Reply:
OK! improved!